### PR TITLE
Review Data Modelling section

### PR DESCRIPTION
**Pull request by hola-soy-milk**

### CODE REVIEW

File: content/2.data-modeling/2.collections.md [hunk 1]
```diff
@@ -162,6 +163,10 @@ When Directus is connected to an existing database, it will introspect existing
 
 To access the collection via the Data Studio, you must configure the collection with additional Directus-specific metadata, such as interfaces, displays, and icons.
 
+::callout{type="tutorials" url="/tutorials/tags/connecting-to-an-existing-database"}
+Read tutorials on adding Directus-specific metadata.
+::
```
Comment (phazonoverload) [inline content/2.data-modeling/2.collections.md]: This doesn't exist yet, so let's leave it out.

File: content/2.data-modeling/3.fields.md [hunk 2]
```diff
@@ -4,20 +4,20 @@ description: Directus fields let you define how your data is stored and displaye
 
 # Fields
 
-Fields are database fields with additional metadata and configuration used by Directus, such as how data is displayed, validations, and interfaces.
+Fields are database columns with additional metadata and configuration used by Directus, such as [how data is displayed](#display), [validations](#validation), and [interfaces](/data-modeling/interfaces).
```
Comment (phazonoverload) [inline content/2.data-modeling/3.fields.md]: Could we please avoid manual in-page links? It makes maintenance harder if we want to change headers later.

**Review by phazonoverload — APPROVED** [review]
Looks good - one small change and then you can merge it.

File: content/2.data-modeling/3.fields.md (hunk 2 above)
Comment (hola-soy-milk) [reply content/2.data-modeling/3.fields.md]: Understood, thanks for the shout out!

File: content/2.data-modeling/2.collections.md (hunk 1 above)
Comment (hola-soy-milk) [reply content/2.data-modeling/2.collections.md]: Got it!

**Comment by hola-soy-milk** [comment]
@phazonoverload Sorted, thank you! Could you please take another look?

**Pull request merged**